src/server/router/users.ts

```diff
@@ -47,9 +47,9 @@
       });
     }),
   removeRole: protectedProcedure
-    .input(z.object({ id: z.string(), role: z.nativeEnum(Role) }))
+    .input(z.object({ id: z.string().cuid().optional(), role: RoleSchema }))
     .mutation(async ({ ctx, input }) => {
-      if (!ctx.session.user.role.includes(Role["ADMIN"])) {
+      if (!ctx.session.user.role.includes(RoleSchema.Enum.ADMIN)) {
         throw new TRPCError({ code: "UNAUTHORIZED" });
```

Comment:
Not an issue. The id is optional as a work around to allow the prisma database to generate the CUIDs for data entries instead of doing that job on the front end. So the key assumption here is that the id will always be available.